app/src/main/java/org/simple/clinic/phone/TwilioMaskedPhoneCaller.kt

```diff
@@ -19,13 +19,11 @@ class TwilioMaskedPhoneCaller @Inject constructor(
 
   private fun maskNumber(config: PhoneNumberMaskerConfig, numberToMask: String): String {
     return if (config.maskingEnabled) {
-      // TODO: This number is temporary. Every nurse will
-      // be assigned a number during login/registration.
-      val twilioNumber = "+1 111 111 1111"
+      val number = "07930684724"
```

Comment:
Instead of hardcoding this number, we should provide it via the `PhoneNumberMaskerConfig`.

Reply:
Extracted this into `PhoneNumberMaskerConfig` as part of a new commit.